packages/taquito-http-utils/src/taquito-http-utils.ts

```diff
@@ -33,29 +35,11 @@ export interface HttpRequestOptions {
   mimeType?: string;
 }
 
-/**
- *  @category Error
- *  @description This error will be thrown when the endpoint returns an HTTP error to the client
- */
-export class HttpResponseError extends Error {
-  public name = 'HttpResponse';
-
-  constructor(
-    public message: string,
-    public status: STATUS_CODE,
-    public statusText: string,
-    public body: string,
-    public url: string
-  ) {
-    super(message);
-  }
-}
-
 /**
  *  @category Error
  *  @description Error that indicates a general failure in making the HTTP request
  */
-export class HttpRequestFailed extends Error {
+export class HttpRequestFailed extends ParameterValidationError {
```

Comment:
I don't think this is a ParameterValidationError, it should fall under a category like 'HttpError'

Reply:
Will create HttpError category and extends from it